addons/website_slides/controllers/thread.py

```diff
@@ -0,0 +1,38 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.addons.mail.controllers import thread
+from odoo import _, http
+from odoo.http import request
+from odoo.exceptions import ValidationError
+
+
+class ThreadController(thread.ThreadController):
+
+    @http.route()
+    def mail_message_post(self, thread_model, thread_id, post_data, context=None, **kwargs):
+        previous_post = False
+        if thread_model == "slide.channel" and post_data.get("subtype_xmlid") == "mail.mt_comment":
+            previous_post = request.env["mail.message"].search(
```

Comment:
In that case, I think `_search` with `limit=1` is even better. We don't need the length of the messages here.

Reply:
`search` (no underscore) with `limit` (and default `order by`) is known to choose terrible plan and to make bad queries (preferring to scan according to order hoping to find one quickly, instead of using search criteria).

And if you actually mean `_search` (with underscore), then it is exactly what `search_count` does.